client/my-sites/site-settings/index.js

```diff
@@ -118,5 +119,13 @@ export default function () {
 	page( '/settings/analytics/:site_id?', redirectToTraffic );
 	page( '/settings/seo/:site_id?', redirectToTraffic );
 
-	page( '/settings/:section', legacyRedirects, siteSelection, sites, makeLayout, clientRender );
+	page(
+		'/settings/:section',
+		legacyRedirects,
+		redirectToGeneral,
```

Comment:
Is it a good idea to move `redirectToGeneral` after `siteSelection`? Just because it seems like we are relying on the site possibly being there already in `redirectToGeneral`.